runner_manager/models/settings.py

```diff
@@ -39,6 +39,7 @@ class Settings(BaseSettings):
     allowed_hosts: Optional[Sequence[str]] = [
         "*",
     ]
+    healthcheck_interval: int = 60
```

Comment:
Not so sure that I'll keep this configuration settings, I migth wait for the work on the group healthcheck.